central/processbaseline/service/service_impl.go

```diff
@@ -172,6 +173,45 @@ func (s *serviceImpl) LockProcessBaselines(ctx context.Context, request *v1.Lock
 	return resp, nil
 }
 
+func (s *serviceImpl) getKeysForNamespaces(ctx context.Context, clusterId string, namespaces []string) ([]*storage.ProcessBaselineKey, error) {
+	query := search.NewQueryBuilder().
+		AddExactMatches(search.Namespace, namespaces...).
+		AddExactMatches(search.ClusterID, clusterId).ProtoQuery()
+
+	baselines, err := s.dataStore.SearchRawProcessBaselines(ctx, query)
```

Comment:
This walks through the table. There is probably a more efficient way of doing this. I am not sure what is the best way of doing this.